src/ex4/server/routes/tasksApi.js

```diff
@@ -13,17 +13,28 @@ router.get("/", async (req, res) => {
 router.post("/", async (req, res) => {
   const taskInput = req.body.itemName;
```

Comment:
you can use speard for nicer syntax:
const {itemName, status, position} = req.body;

you can also give different naming if you want:
const {itemName: taskInput, status: isCompleted, position} = req.body;